CMakeLists.txt

```diff
@@ -171,39 +173,49 @@ function(add_valgrind_test NAME_WITH_PREFIX BINARY WORKING_DIRECTORY)
     endif()
 endfunction()
 
-get_target_property(QMAKE Qt::qmake IMPORTED_LOCATION)
-set(CARGO_ENV "QMAKE=set:${QMAKE}")
-set(RUNTIME_ENV "")
-
-# On windows, Qt dll needs to be in the PATH for the tests to run
-if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
-    execute_process(
-        COMMAND ${QMAKE} -query QT_INSTALL_BINS
-        OUTPUT_VARIABLE QT_INSTALL_BINS
-        OUTPUT_STRIP_TRAILING_WHITESPACE
-    )
-
-    execute_process(
-        COMMAND ${QMAKE} -query QT_INSTALL_PLUGINS
-        OUTPUT_VARIABLE QT_INSTALL_PLUGINS
-        OUTPUT_STRIP_TRAILING_WHITESPACE
-    )
-
-    execute_process(
-        COMMAND ${QMAKE} -query QT_INSTALL_QML
-        OUTPUT_VARIABLE QT_INSTALL_QML
-        OUTPUT_STRIP_TRAILING_WHITESPACE
-    )
-
-    list(
-        APPEND
-        RUNTIME_ENV
-        "PATH=path_list_append:${QT_INSTALL_BINS}"
-        "QT_PLUGIN_PATH=path_list_append:${QT_INSTALL_PLUGINS}"
-        "QML_IMPORT_PATH=path_list_append:${QT_INSTALL_QML}"
-        "QML2_IMPORT_PATH=path_list_append:${QT_INSTALL_QML}"
-    )
-    list(APPEND CARGO_ENV ${RUNTIME_ENV})
+if(BUILD_WASM)
```

Comment:
did `if(EMSCRIPTEN)` not work?

Reply:
you get that if you use `emcmake cmake` or `emconfigure cmake`